continuous-testing/starter-kits/README.md

```diff
@@ -1,3 +1,54 @@
 # Starter Kits
 
-Nothing here yet, come back soon... 
\ No newline at end of file
+This page contains starter kit information, which represent templates, code and configuration to help you get started quickly with continuous testing best practices described in this overall guide. Please see categories and links below for details. 
+
+## Software Composition Analysis
+
+This section contains links to sample actions, templates and configurations that analyze and validate composition of Open Source Software (OSS) components in software systems. Identifying software and licensing vulnerabilites and ensuring routine software updates is an [OSS cybersecurity](https://www.cisa.gov/uscert/ncas/alerts/aa22-137a) [best practice](https://appel.nasa.gov/2022/06/30/spotlight-on-lessons-learned-open-source-and-commercial-web-software-vulnerabilities/).  
+
+### Dependabot
+
```

Comment:
```suggestion
A GitHub ecosystem tool for dependency version and security vulnerability analysis. 
```

Might be good to be clear this is only for GitHub, not any other VCS.

Reply:
Added inline to document.